pkg/providers/v1/instances_v2.go

```diff
@@ -74,9 +79,32 @@ func (c *Cloud) getAdditionalLabels(zoneName string) (map[string]string, error)
 
 	additionalLabels[LabelZoneID] = zoneID
 
+	if c.topologySupportedInstance(instanceType) && slices.Contains(c.cfg.Global.TopologySupportedRegions, region) {
+		topologyRequest := &ec2.DescribeInstanceTopologyInput{InstanceIds: []*string{&instanceID}}
+		topology, err := c.ec2.DescribeInstanceTopology(topologyRequest)
+		if err != nil || topology == nil {
```

Comment:
I revised the topologySupportedInstance to check the supported instance types instead of the instance prefix, so the DescribeInstanceTopology api should supported here. I will throw error directly